modes/vterm/evil-collection-vterm.el

```diff
@@ -101,7 +153,22 @@ also uses `evil-mode'."
     "[[" 'vterm-previous-prompt
     "]]" 'vterm-next-prompt
     "p" 'vterm-yank
-    "u" 'vterm-undo))
-
+    "a" 'vterm-evil-append
+    "A" 'vterm-evil-append-line
+    ;; "d" 'vterm-evil-delete
```

Comment:
Would be cool if these vterm functions words, etc e.g. dw, dW etc etc, but that's probably out of scope.

These binds look ok to me.